components/backup/src/endpoint.rs

```diff
@@ -382,7 +382,7 @@ impl<E: Engine, R: RegionInfoProvider> Endpoint<E, R> {
                 let key = brange
                     .start_key
                     .clone()
-                    .map(|k| hex::encode(k.into_raw().unwrap()));
+                    .and_then(|k| tikv_util::file::sha256(&k.into_raw().unwrap()).ok());
```

Comment:
Could you add some comments about why we need it?

Reply:
sure, add [comment](https://github.com/tikv/tikv/pull/6177/files#diff-53831868d19a8c3495a300c681e252a8R383)